public/js/app/components/statistics/AnalysesSummary.jsx

```diff
@@ -25,18 +25,30 @@ function addAllSchoolsData({ totalCourses, totalUniqPublishedAnalyses }) {
   return allSchools
 }
 
-function Captions({ year, seasons }) {
-  const { translation, languageIndex } = useLanguage()
+function Captions({ school, year, seasons }) {
+  const {
+    languageIndex,
+    translation: {
+      statisticsLabels,
+      statisticsLabels: {
+        formLabels: { formSubHeaders },
+      },
+    },
+  } = useLanguage()
 
   const seasonsStr = seasons.map(season => seasonLib.labelSeason(season, languageIndex)).join(', ')
   return (
     <Row>
       <Col xs="4" style={{ flex: 'none', width: 'auto', paddingBottom: '20px' }}>
-        <label>{translation.statisticsLabels.formLabels.formSubHeaders.year}</label>
+        <label>{formSubHeaders.school}</label>
+        {`: ${school === 'allSchools' ? statisticsLabels[school] : school}`}
```

Comment:
I always have a hard time reading ternary operators inline like that. What do you think of extracting that part to something like `getSchoolName` or something like that?

Reply:
We liked the idea and we made the required changes 🚀